### PR TITLE
Docs: Design guide

### PR DESCRIPTION
**Pull request by ericfennis**
Add some guidelines to help people to design icons.

### CODE REVIEW

File: docs/ICON_DESIGN_GUIDE.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Icon Degin Guide
```
Comment (locness3) [inline docs/ICON_DESIGN_GUIDE.md:1]:
```suggestion
# Icon Design Guide
```

File: docs/ICON_DESIGN_GUIDE.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+To design icons for this icon library we need to follow some guidelines to match all the icons in the same style.
```
Comment (locness3) [inline docs/ICON_DESIGN_GUIDE.md:3]:
```suggestion
Here are rules that should be followed to keep quality and consistency when making icons for Lucide.
```

File: docs/ICON_DESIGN_GUIDE.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+1. Each Icon is desined on a canvas of **24 by 24 pixels**.
+2. Each Icon have **1 pixel padding** between icon and the canvas.
+3. Each icon has a **stroke width of 2 pixels**.
+4. Each icon has **round joins**.
+5. Each icon has **round caps**.
+6. Each icon has **centered strokes**.
+7. Each icon containing squares has a **border radius of 2 pixels**.
+8. Each icon **2 pixels of spacing between elements**.
+
+## The Rules Visualized
```
Comment (locness3) [inline docs/ICON_DESIGN_GUIDE.md:16]: I'd remove that and directly have the rules.

**Review by locness3 — CHANGES_REQUESTED** [review]
Also, the images don't seem to work.

File: docs/ICON_DESIGN_GUIDE.md (hunk 3 above)
Comment (ericfennis) [reply docs/ICON_DESIGN_GUIDE.md:16]: Yeah with the images, it not directly clear what the rules are. So thats why I added a summary

**Comment by ericfennis** [comment]
Thanks for your review @locness3 

I don't know why the images are not wokring in the PR file viewer.
But if you look [here](https://github.com/lucide-icons/lucide/blob/2192993b14c8d6c7951c296bad5a8e494852520e/docs/ICON_DESIGN_GUIDE.md), you can see them.

**Pull request merged**